src/concurrency.md

```diff
@@ -97,7 +98,7 @@ fn main() {
     // [2]
     vec.par_iter_mut().for_each(|p| {
         // [3]
-        *p = rand::weak_rng().gen_ascii_chars().take(5).collect()
+        *p = (0..5).map(|_| thread_rng().sample(Alphanumeric)).collect();
```

Comment:
This creates a new `Rc` reference for each char. Still, maybe we can optimise that away.